### ISSUE TITLE
Icon edit in STYLER (up/down click)

### ISSUE THREAD

**Issue by martdog** [Teemill/TMLL#55]
Potential bug, but may have been slow connection..
- couldn't see the icon move when we clicked up and down on the arrows
-also worth making each click move it 5px not 1px.

**Comment by G-Bro** [comment]
@martdog I believe this is fixed. Are you talking about the logo margin top input box?
Have upped increments to 5 per click.

**Comment by G-Bro** [comment]
@martdog could you please clarify above

**Comment by martdog** [comment]
yes upped is good, thanks. I think I may have been on a slow connection: I could not see the icon size move on edge browser. Luckily, there a surface in the bag by whiteboard, 2526
